database/migrations/2019_06_21_204239_comments.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class Comments extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('comments', function (Blueprint $table) {
+
+            $table->increments('id');
+            $table->string('text');
+            $table->boolean("is_secret");
+            $table->string('entity_type');
+            $table->integer('entity_id')->unsigned();
+            $table->integer('user_id')->unsigned();
+
+            $table->timestamps();
+
+            $table->foreign('user_id')
+                ->references('id')
+                ->on('users')
+                ->onDelete('cascade');
+        });
+
+        DB::statement("INSERT INTO users (id, username, password, profile_id, profile_type, is_admin) VALUES ('0', 'SYS', '', 0, 'App\\\\Member', true)");
+
+        DB::statement("
+            INSERT INTO comments (text, entity_type, entity_id, is_secret, user_id)
+                SELECT * FROM (
+                    SELECT opmerkingen_admin, 'App\\\\Member', id, true, 0
```

Comment:
```suggestion
                    SELECT opmerkingen_admin, 'App\\\\Member', id, false, 0
```

Toch? Gezien alleen de geheime opmerkingen geheim zijn. Wil je aan de hand hiervan ook even meekijken of alles in `down()` migration wel goed gaat?